README.md

```diff
@@ -890,6 +890,7 @@ Table of Contents
   * [Bunny](https://bunny.net)
     * [Bunny Fonts](https://fonts.bunny.net/) - All the Google Fonts with Google Fonts drop-in compatible API. Privacy oriented!.
     * [Bunny DNS](https://bunny.net/dns/) - DNS hosting, 20 million free queries
+  * [FontBolt](https://www.fontbolt.com/) - Large database of pop culture fonts
```

Comment:
```
You are hereby granted a non-exclusive and limited license to use the fonts available on this website for personal and commercial use or demo. All commercial use is subject to getting a license from the creator/owner of the font. Thus, if a font is tagged on our website as “personal use”, then you must simply use such fonts for personal purposes. If the font is tagged as commercial, then you may ONLY use the font commercially once you have received a license from its owner. 
``` 

Think I'll pass on this one, it's just awkward

Thank you for the contribution though